rmrl/__main__.py

```diff
@@ -29,9 +30,16 @@ def main():
     parser.add_argument('--alpha', default=0.3, help="Opacity for template background (0 for no background).")
     parser.add_argument('--no-expand', action='store_true', help="Don't expand pages to margins on device.")
     parser.add_argument('--only-annotated', action='store_true', help="Only render pages with annotations.")
+    parser.add_argument('--black', default='#000000', help="Color for \"black\" pen, in hex notation (#RRGGBB).")
+    parser.add_argument('--white', default='#FFFFFF', help="Color for \"white\" pen, in hex notation (#RRGGBB).")
     parser.add_argument('--version', action='version', version=VERSION)
     args = parser.parse_args()
 
+    for color, value in [('White', args.white), ('Black', args.black)]:
+        if re.search(r'^#[0-9a-f]{6}$', value, re.I) is None:
```

Comment:
Instead of looking at a form, let's just try parsing the input into a color and aborting if that fails.  That way, we can improve the color parsing code without need to update this code as well.

Reply:
Sure.  The new code just passes the strings to `render()` as-is.  `render()` does the parsing and raises an `InvalidColor` exception if the parsing fails.